core/src/main/web/outputdisplay/bko-plot/bko-plot.js

```diff
@@ -544,17 +544,25 @@
           scope.jqcontainer.find("#legends").remove();
 
           scope.legendDone = true;
-          var legend = $("<table></table>").appendTo(scope.jqcontainer)
-            .attr("id", "legends")
-            .attr("class", "plot-legendcontainer")
+          var legendScrollableContainer = $("<div></div>").appendTo(scope.jqcontainer)
+            .attr("class", "plot-legendscrollablecontainer")
             .draggable({
               stop : function(event, ui) {
                 scope.legendPosition = {
                   "left" : ui.position.left,
                   "top" : ui.position.top
                 };
-              }
-            });
+              },
+              handle : "#legendDraggableContainer"
+            })
+            .css("max-height", scope.jqsvg.height() - scope.layout.bottomLayoutMargin);
+
+          var legendDraggableContainer = $("<div></div>").appendTo(legendScrollableContainer)
+            .attr("id", "legendDraggableContainer")
```

Comment:
You mean #legendDraggableContainer? jQuery draggable.handle works only with elements that descend from the draggable element. I've tested 2 plots with legend on the same page – legends are ok